### DISCUSSION TITLE
Generic vs Specific

### DISCUSSION THREAD

**Discussion by katastrophe92** [zen-browser/desktop#1442] [Q&A]
Hello Zen team and community!
I have only one question. What is the difference between Generic version and Specific version for Windows?
Hope someone can answer my question. Thanks!

**Comment by mr-cheffy** [comment]
Specific uses certain instruction optimizations like AVX2. This is for faster execution but less CPU support it.

Checkout if your CPU is supported here https://docs.zen-browser.app/guides/generic-optimized
